sympy/assumptions/assume.py

```diff
@@ -86,80 +131,217 @@ def arg(self):
         x + 1
 
         """
-        return self._args[1]
+        args = self.args
+        if len(args) == 1:
+            return args[0]
+        return args
 
     @property
     def args(self):
-        return self._args[1:]
+        return self._args[1]
 
     @property
     def func(self):
         return self._args[0]
 
     @cacheit
     def sort_key(self, order=None):
-        return (self.class_key(), (2, (self.func.name, self.arg.sort_key())),
+        return (self.class_key(),
+                (2, (self.func.sort_key(), self.arg.sort_key())),
                 S.One.sort_key(), S.One)
```

Comment:
Why is a special `sort_key` method needed? Can we not just use the `Basic.sort_key` method?

Reply:
I think we can.